### ISSUE TITLE
Bundler issue with the logical nullish assignment (x ??= y) operator

### ISSUE THREAD

**Issue by jveres** [denoland/deno#8627]
Deno version 1.5.4

```sh
echo "let a = null, b = 2; console.log(a ??= b);" > test.ts
deno run test.ts
2
deno bundle test.ts test.bundle.js
deno run test.bundle.js
error: Unexpected token `=`. Expected this, import, async, function, [ for array literal, { for object literal, @ for decorator, function, class, null, true, false, number, bigint, string, regexp, ` for template literal, (, or an identifier at file:///Users/U715864/temp/test.bundle.js:2:16
```
cc @kdy1

**Comment by kdy1** [comment]
@kitsonk This seems like a bug of `deno run`. It works well with fixture testing, but it errors on `deno run`.

**Comment by kdy1** [comment]
Ah.. I found the root cause.
`??=` is typescript-only extension.

What should we do here? Transform it from typescript::strip?

**Comment by kitsonk** [comment]
@kdy1 no it is Stage 4 proposal for ECMAScript: https://github.com/tc39/proposal-logical-assignment.  It is valid JavaScript in most greenfield browsers: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_nullish_assignment

**Comment by kdy1** [comment]
Oh, thanks! I didn't know it. Then I'll simply allow `??=` in js files.

**Comment by kitsonk** [comment]
It is also `||=` and `&&=` that are being added.

**Comment by jveres** [comment]
Fixed in `deno v1.6.2`.
